app/models/form/sales/questions/living_before_purchase.rb

```diff
@@ -1,15 +1,26 @@
 class Form::Sales::Questions::LivingBeforePurchase < ::Form::Question
   def initialize(id, hsh, page)
     super
-    @id = "proplen"
-    @check_answer_label = "Number of years living in the property before purchase"
-    @header = "How long did the buyer(s) live in the property before purchase?"
-    @hint_text = "You should round this up to the nearest year. If the buyers haven't been living in the property, enter '0'"
-    @type = "numeric"
-    @min = 0
-    @max = 80
-    @step = 1
-    @width = 5
-    @suffix = " years"
+    @id = "proplen_asked"
```

Comment:
Can we have _known to align with the rest of the questions, or is this a field expected in exports?